lib/trento/infrastructure/software_updates/adapter/mock_suma.ex

```diff
@@ -59,7 +59,7 @@ defmodule Trento.Infrastructure.SoftwareUpdates.MockSuma do
            to_version: "1.16.2",
            to_release: "1",
            to_epoch: "0",
-           to_package_id: 92_348_112_636
+           to_package_id: "92348112636"
```

Comment:
This change will require to reset SUMA values in demo, as using an integer was not expected

Reply:
@nelsonkopliku @janvhs Do you know why we moved this value to integer in the past?
Here: https://github.com/trento-project/web/pull/2916
The `to_package_id` value is defined as string in the SUMA docs: https://documentation.suse.com/suma/4.3/api/suse-manager/api/system.html#apidoc-system-listLatestUpgradablePackages-loggedInUser-sid

With that change, if the value is really a string I don't really know how this worked in the first place...
And if it is an number, our backend code was wrong.

Reply:
not sure why it became an integer. In other places is considered a string, as it should be.
What happens if you just put back to strings in the mock definition?

Reply:
Well, it works with my mocked data. I was wondering if the SUMA docs have some typo.
I will start a SUMA instance and try myself.
The code with different types is suspicious

Reply:
True, it looks like suma is actually returning an integer, at least on our testing instance running on 4.3

Reply:
Yes. I'm going to change the field type to integer. Now everything works with a real SUMA instance.
I asked the SUMA team as well. 
I hope it is only a documentation error, otherwise we might need to do changes if they fix it